packages/gdb/gdb.0.3/opam

```diff
@@ -26,4 +26,4 @@ depends: [
   "ocamlbuild" {build}
 ]
 depopts: ["lambda-term" {build}]
-available: [ocaml-version >= "4.02.0"]
+available: [ocaml-version >= "4.02.0" & ocam-version < "4.03.0"]
```

Comment:
@avsm, @damiendoligez: that should be `ocaml-version` rather than `ocam-version`...